strapi/strapi
# fix(Content-releases): timezone null for timezones UTC-HH:MM

### What does it do?

- Timezone value was sending null for `UTC-HH:MM`, so timezone value concatenated with `_` instead of `-`
- Scheduling check handler restricted to the field and not full width.

### How to test it?

- Please create a new release for `UTC-HH:MM`  and should be able to see the scheduling info on release edit page.
- Scheduling check handler should be restricted to the field and not full width.
